.github/CODEOWNERS

```diff
@@ -45,6 +45,7 @@ exporter/googlemanagedprometheusexporter/            @open-telemetry/collector-c
 exporter/googlecloudpubsubexporter/                  @open-telemetry/collector-contrib-approvers @alexvanboxel
 exporter/humioexporter/                              @open-telemetry/collector-contrib-approvers @xitric
 exporter/influxdbexporter/                           @open-telemetry/collector-contrib-approvers @jacobmarble
+exporter/instanaexporter/                            @open-telemetry/collector-contrib-approvers @hickeyma
```

Comment:
Given I'm the sponsor, I believe I should be part of the code owners for this component.